### PR TITLE
Merchant box deletion

### PR DESCRIPTION
**Pull request by Nerrathiel**
Merchant code machine broke.

This fixes it.

No more buying boxes for ~~40 creds and getting all the shit inside for free.

### CODE REVIEW

**Comment by Nerrathiel** [comment]
!review

File: .gitignore [hunk 1]
```diff
@@ -1,19 +0,0 @@
-#ignore misc BYOND files
```
Comment (Arrow768) [inline .gitignore]: Why did you delete the .gitignore?

File: html/changelogs/Naelynn-PR-6902.yml [hunk 2]
```diff
@@ -1,42 +1,41 @@
-################################
```
Comment (Arrow768) [inline html/changelogs/Naelynn-PR-6902.yml]: Removes the example changelog

Comment (Nerrathiel) [reply html/changelogs/Naelynn-PR-6902.yml]: when the fck was this done... I didn't mean to touch example changelog x_X

File: .gitignore (hunk 1 above)
Comment (Nerrathiel) [reply .gitignore]: I thought it's unwanted files...

File: html/changelogs/Naelynn-PR-6902.yml (hunk 2 above)
Comment (Nerrathiel) [reply html/changelogs/Naelynn-PR-6902.yml]: It should be back now

File: .gitignore (hunk 1 above)
Comment (Nerrathiel) [reply .gitignore]: Should be back now

File: html/changelogs/Naelynn-PR-6902.yml (hunk 2 above)
Comment (Arrow768) [reply html/changelogs/Naelynn-PR-6902.yml]: Nope that's not the example changelog you readded

File: .gitignore [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
+cfg/
+build_log.txt
+
+__pycache__/
+*.py[cod]
+*$py.class
+/.atom-build.json
+.vscode/
+.vs/
+html/changelogs/example - Copy.yml
```
Comment (Sindorman) [inline .gitignore]: This wasn't here before

Comment (Nerrathiel) [reply .gitignore]: Thank you for catching it! Should be fixed!

File: html/changelogs/Naelynn-PR-6902.yml (hunk 2 above)
Comment (Nerrathiel) [reply html/changelogs/Naelynn-PR-6902.yml]: I genuinely have no idea where that came from. Should be fixed now. Thank you for checking!
